Anlab.Mvc/Views/TestItems/Create.cshtml

```diff
@@ -0,0 +1,39 @@
+@model Anlab.Core.Domain.TestItem
+
+<!DOCTYPE html>
+
+<html>
+<head>
+    <meta name="viewport" content="width=device-width" />
+    <title>Create</title>
+</head>
+<body>
```

Comment:
@laholstege We never do this.
Basically, just do this instead:
@{
    ViewData["Title"] = "Create";
}